tools/pepsi/src/upload.rs

```diff
@@ -38,6 +38,40 @@ pub struct Arguments {
     /// The NAOs to upload to e.g. 20w or 10.1.24.22
     #[arg(required = true)]
     pub naos: Vec<NaoAddress>,
+    #[arg(long)]
+    pub skip_check_os: bool,
+}
+
+pub async fn check_os(nao: &Nao) -> bool {
```

Comment:
This should live in the `nao` crate. Then you can also revert the `pub` changes. The idea of the `nao` crate is to hide all low-level SSH stuff but exposed that now. 😉

Reply:
Please also revert the `pub` changes.